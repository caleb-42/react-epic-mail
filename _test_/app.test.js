import React from "react";
import renderer from "react-test-renderer";
import App from "../src/app";

it("should output Hello", () => {
  /* const tree = renderer.create(<App />);
  expect(tree).toMatchSnapshot(); */
});
