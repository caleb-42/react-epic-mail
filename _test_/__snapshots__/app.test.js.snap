// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should output Hello 1`] = `
<h1
  className="landing"
>
  Welcom to EPIC MAIL
</h1>
`;
